pkg/admin/graphql/user_mutation.go

```diff
@@ -164,6 +164,77 @@ var _ = registerMutationField(
 	},
 )
 
+var markPasswordAsExpiredInput = graphql.NewInputObject(graphql.InputObjectConfig{
+	Name: "MarkPasswordAsExpiredInput",
```

Comment:
According to the spec, the name is SetPasswordExpiredBlahBlahBlah

Reply:
It was updated in later commit.